openshift_tools/cloud/aws/ebs_snapshotter.py

```diff
@@ -370,24 +389,25 @@ def trim_snapshots(self, hourly_backups, daily_backups, weekly_backups, monthly_
             all_expired_snapshots.extend(expired_snapshots)
 
             self.print_snapshots(expired_snapshots, \
-                                 msg="Expired Snapshots (%d):" % len(expired_snapshots), prefix="      ")
+                                 msg="Expired Snapshots (%d):" % len(expired_snapshots), prefix=" " * 6)
             self.verbose_print()
 
-            self.verbose_print("Removing snapshots (%d):" % len(expired_snapshots), prefix="      ")
+            self.verbose_print("Removing snapshots (%d):" % len(expired_snapshots), prefix=" " * 6)
 
             # actually remove expired snapshots
             for exp_snap in expired_snapshots:
-                self.verbose_print(exp_snap.id, prefix="        ")
+                self.verbose_print(exp_snap.id, prefix=" " * 8)
                 try:
                     if dry_run:
-                        self.print_dry_run_msg(prefix="          ")
+                        self.print_dry_run_msg(prefix=" " * 10)
                     else:
                         exp_snap.delete()
                         deleted_snapshots.append(exp_snap)
                 # Reason: disable pylint broad-except because we want to continue on error.
                 # Status: permanently disabled
                 # pylint: disable=broad-except
                 except Exception as ex:
+                    # pylint: disable=no-member
```

Comment:
Please add a reason block here per our best practices:
https://github.com/openshift/openshift-tools/blob/prod/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code